docs/content/cookbook/oss-deployment-guide.md

```diff
@@ -0,0 +1,625 @@
+
+```bash
+certbot --nginx
+``` 
+
+when certbot is done, your nginx config file will look something like this:
+```nginx
+   server {
+       
+       server_name     shoppernauts.com;    # setup your domain here
+   
+       gzip            on;
+       gzip_types      text/plain application/xml text/css application/   javascript;
+       gzip_min_length 1000;
+   
+       location / {
+          # expires $expires;
+   
+           proxy_redirect                      off;
+           proxy_set_header Host               $host;
+           proxy_set_header X-Real-IP          $remote_addr;
+           proxy_set_header X-Forwarded-For    $proxy_add_x_forwarded_for;
+           proxy_set_header X-Forwarded-Proto  $scheme;
+           proxy_read_timeout          1m;
+           proxy_connect_timeout       1m;
+           proxy_pass                          http://127.0.0.1:3000; # set the address of your frontend instance here
+       }
+   
+       location /api {
+          # expires $expires;
+           rewrite ^/api(.*)$ $1 break;  # Remove the '/api' prefix from the URI
+   
+           proxy_redirect                      off;
+           proxy_set_header Host               $host;
+           proxy_set_header X-Real-IP          $remote_addr;
+           proxy_set_header X-Forwarded-For    $proxy_add_x_forwarded_for;
+           proxy_set_header X-Forwarded-Proto  $scheme;
+           proxy_read_timeout          1m;
+           proxy_connect_timeout       1m;
+           proxy_pass                          http://127.0.0.1:4000; # set the address of api instance here
+       }
+   
+   
+   
+   
+       listen 443 ssl; # managed by Certbot
+       ssl_certificate /etc/letsencrypt/live/shoppernauts.com/fullchain.pem; #    managed by Certbot
+       ssl_certificate_key /etc/letsencrypt/live/shoppernauts.com/privkey.pem; #    managed by Certbot
+       include /etc/letsencrypt/options-ssl-nginx.conf; # managed by Certbot
+       ssl_dhparam /etc/letsencrypt/ssl-dhparams.pem; # managed by Certbot
+   
+   }
+   server {
+       if ($host = shoppernauts.com) {
+           return 301 https://$host$request_uri;
+       } # managed by Certbot
+   
+   
+       listen                80;
+       
+       server_name     shoppernauts.com;
+       return 404; # managed by Certbot
+   
+   
+}
+```
+
+## Setting up Services
+### Overview
+`systemctl` is a command-line tool that allows for the management and monitoring of the `systemd` system and service manager. It is used to start, stop, and restart services, as well as to configure services to automatically start when the system boots. We'll use it to start our frontend and backend applications. To do this, we'll first create a service for each application.
+
+### Create a service for the frontend
+Create a file called vsfcommunity.service in `/etc/systemd/system/` and paste the following contents into it:
```

Comment:
Not sure if this is intentional, but this doesn't match the example repo you gave. Not a big deal if this whole guide is built on these namings though.